### ISSUE TITLE
BUG - Compatibility issues with Forien's Quests

### ISSUE THREAD

**Issue by Jonwh25** [Sigafoos/journal-links#8]
When linking back to a quest entry it goes to the actual quest journal.  Dunno if that is a fixable thing or not due to the way your system works but thought I'd point it out.

**Comment by Sigafoos** [comment]
Thanks for the report! I’ll definitely look into it, as these seem like modules people would want together. 

I’m going to try to sort out #7 first though.

**Comment by scooper4711** [comment]
Yes - this is critical. I don't want the players to be able to open the underlying journal entry. There be secrets there.
